test/unit/NodeUpdateTransaction.js

```diff
@@ -166,6 +166,7 @@ describe("NodeUpdateTransaction", function () {
 
     describe("frozen transaction", function () {
         let tx;
+
         before(function () {
             const ACCOUNT_ID = AccountId.fromString("0.4.20");
```

Comment:
Change all similar strings.

Reply:
why?

Reply:
Oh "strings", I read "things". 

Ah, it's going to cost some time and the string id doesn't matter at all. 

I don't see the need to put in time for it. No need to be all corporate, we can keep some internet slang in it. Also similar strings are incorporated in different SDKs.